src/App.jsx

```diff
@@ -1,14 +1,28 @@
 import React from 'react';
+import { Sum } from './components/Sum';
 import './App.css';
 
 export const App = () => (
   <>
-    <p>Sum of 2 and 3 is 5</p>
-    <p>Sum of -5 and 5 is 0</p>
-    <p>Sum of 10 and 0 is 10</p>
-    <p>Sum of 0 and 5 is 5</p>
-    <p>Sum of 0 and 0 is 0</p>
-    {/* Replace paragraphs with Sum componets */}
-    {/* And remove commented lines :) */}
+    <Sum
+      a={2}
+      b={3}
+    />
+    <Sum
+      a={-5}
+      b={5}
+    />
+    <Sum
+      a={10}
+      b={0}
+    />
+    <Sum
+      a={0}
+      b={5}
+    />
+    <Sum
+      a={0}
+      b={0}
+    />
```

Comment:
![image](https://user-images.githubusercontent.com/59209337/213106449-55751b8f-5011-4c4f-8e19-e4ee45c52881.png)

should be like in the task